pkg/common/parameters.go

```diff
@@ -134,6 +134,7 @@ type ParameterProcessor struct {
 type ModifyVolumeParameters struct {
 	IOPS       *int64
 	Throughput *int64
+	SizeGb     *int64
```

Comment:
As discussed offline, doing a size change in modify volume will cause problems because there's no node rpc to update the filesystem?

Reply:
Oh, I see, you're just using this in controller expand volume.

I think we at least need a comment that this is only used in that RPC so there's no future confusion.

Reply:
I also need to change the current implementation of the GCE Update Disk, there is a restriction right now that the request has to pass in one of iops or throughput, while it can take sizeGb only. And we will move the iops/throughput check in the actual ControllerExpandVolume func.

Reply:
Hmm it turns out to be more complicated, because currently we have resizeZonalDisk + resizeRegionalDisk, but for update we only have updateZonalDisk.